tests/test_rigid_physics.py

```diff
@@ -2751,3 +2751,101 @@ def test_contype_conaffinity(show_viewer, tol):
     assert_allclose(box1.get_pos(), np.array([0.0, 0.0, 0.25]), atol=1e-3)
     assert_allclose(box2.get_pos(), np.array([0.0, 0.0, 0.75]), atol=1e-3)
     assert_allclose(box3.get_pos(), np.array([0.0, 0.0, 0.75]), atol=1e-3)
+
+
+@pytest.mark.parametrize("backend", [gs.cpu])
+def test_examples_api(show_viewer, tol):
```

Comment:
This unit test is not great, there is too much noise and the name is not clear. It should fit in 10 lines at most.